packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -665,6 +665,13 @@ export class SchemaFactory<
 		return new FieldSchema(FieldKind.Optional, t);
 	}
 
+	/**
+	 * Make a field of type identifier instead of the default which is required.
+	 */
+	public identifier(): FieldSchema<FieldKind.Identifier> {
```

Comment:
```suggestion
	public get identifier(): FieldSchema<FieldKind.Identifier> {
```
The spec calls for `sf.identifier` rather than `sf.identifier()`

Reply:
updated with suggestion